Stylesheets/Components/card.css

```diff
@@ -0,0 +1,108 @@
+img {
```

Comment:
You can add a class to the image as card-img . No need of selecting the element related to it.So you can take care of it appropriately.

Reply:
Oh understood!